v3/plugins/joaompinto/asciidoctor-vscode/latest/meta.yaml

```diff
@@ -0,0 +1,12 @@
+publisher: joaompinto
+apiVersion: v2
+id: joaompinto.asciidoctor-vscode
+version: 2.7.6
```

Comment:
I think this meta yaml will fail validation. Here should be version latest: https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/eclipse/che-theia/latest/meta.yaml#L4

Reply:
Good catch. Thanks. Done